src/font7seg.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is a part of SegM8 library.
+ *
+ * Product page: https://amperka.ru/product/SegM8
+ * © Amperka LLC (https://amperka.com, dev@amperka.com)
+ * 
+ * Author: Yury Botov <by@amperka.com>
+ * License: GPLv3, all text here must be included in any redistribution.
+ */
+
```

Comment:
This important information? May be move the information about the author to the library.properties file and all the rest to remove.

Reply:
In all the previous libraries it was like this. However, we will find out the opinion of Victor.

Reply:
http://wiki.amperka.ru/hq:rnd:code:banner